libvmi/Makefile.am

```diff
@@ -2,6 +2,7 @@ SUBDIRS = config
 
 h_public =  libvmi.h \
             libvmi_extra.h \
+			slat.h \
```

Comment:
Actually yes (unintentionally).
My IDE seems to be forgetting that it should use 4 spaces instead of tabs.